service/frontend/workflowHandler.go

```diff
@@ -3706,6 +3717,69 @@ func isFailoverRequest(updateRequest *gen.UpdateDomainRequest) bool {
 	return updateRequest.ReplicationConfiguration != nil && updateRequest.ReplicationConfiguration.ActiveClusterName != nil
 }
 
+func isGraceFailoverRequest(updateRequest *gen.UpdateDomainRequest) bool {
+	return updateRequest.IsSetFailoverTimeoutInSeconds()
```

Comment:
What if someone calls the API with failover timeout set but there's no change in the active cluster field or leaves it as empty?

Reply:
Those will handle in the domain handler. 
First scenario: added in handler to ban this case.
Second scenario: It already ban in the handler